en/docs/install-and-setup/perfromance-tuning-and-test-results/tuning-performance.md

```diff
@@ -1,6 +1,6 @@
 # Tuning Performance
 
-This section describes some recommended performance tuning configurations to optimize the WSO2 API Manager. It assumes that you have set up the API Manager on Unix/Linux, which is recommended for a production deployment. We also recommend [a distributed API Manager setup]({{base_path}}/install-and-setup/deploying-wso2-api-manager/deployment-patterns/) for most production systems. Out of all components of an API Manager distributed setup, the API Gateway is the most critical, because it handles all inbound calls to APIs. Therefore, we recommend you to have at least a 2-node cluster of API Gateways in a distributed setup.
+This section describes some recommended performance tuning configurations to optimize WSO2 API Manager. It assumes that you have set up the API Manager on Unix/Linux, which is recommended for a production deployment. WSO2 also recommends [a distributed API Manager setup]({{base_path}}/install-and-setup/deploying-wso2-api-manager/deployment-patterns/) for most production systems. Out of all components of an API Manager distributed setup, the API Gateway is the most critical, because it handles all inbound calls to APIs. Therefore, WSO2 recommends that you have at least a 2-node cluster of API Gateways in a distributed setup.
```

Comment:
Why don't we have these as points listed under a topic "Recommendations" ?

Reply:
There are only 2 point under the initial paragraph so it may not make sense to have a section saying recommendations, because after all the entire doc covers recommendations for performance tuning. 

@chamilaadhi / @bhathiya , what do you think?

Reply:
agree with @Mariangela

Reply:
Removed the recommendations in the https://github.com/wso2/docs-apim/pull/798/commits/7554bc2eb4e58f59fa40bf8df0f43fe9e5b78b7b fix and I also created https://github.com/wso2/docs-apim/issues/801 based on the discussion with Bhathiya.